python/main.py

```diff
@@ -39,13 +39,17 @@ def root():
 def add_item(name: str = Form(...), category: str = Form(...), image: str = Form(...)):
     conn = sqlite3.connect(data_base_name)
     cur = conn.cursor()
-
+    if cur.fetchone[0]() == 0:
```

Comment:
ここでエラーが返ってきます。
```
File "./main.py", line 43, in add_item
    if cur.fetchone()[0] == 0:
TypeError: 'NoneType' object is not subscriptable
```

```suggestion
    if cur.fetchone() == None:
```
に修正が必要だと思います！